test/app-luatest/gh_6200_lua_cjson_escapes_slash_test.lua

```diff
@@ -0,0 +1,104 @@
+local compat = require('compat')
+local misc = require('test.luatest_helpers.misc')
+local jit = require('jit')
+local json = require('json')
+local popen = require('popen')
+local clock = require('clock')
+local t = require('luatest')
+local g = t.group()
+
+g.test_json_encode = function()
+    compat.json_escape_forward_slash = 'old'
+    -- Test that '/' is escaped with 'old' setting.
+    t.assert_equals(json.encode({url = 'https://srv:7777'}), '{"url":"https:\\/\\/srv:7777"}')
+    t.assert_equals(json.encode('/home/user/tarantool'), [["\/home\/user\/tarantool"]])
+    -- Test that other escape symbols are not affected by the setting.
+    t.assert_equals(json.encode('\t'), [["\t"]])
+    t.assert_equals(json.encode('\\'), [["\\"]])
+
+    compat.json_escape_forward_slash = 'new'
+    -- Test that '/' is not escaped with 'new' setting.
+    t.assert_equals(json.encode({url = 'https://srv:7777'}), [[{"url":"https://srv:7777"}]])
+    t.assert_equals(json.encode('/home/user/tarantool'), [["/home/user/tarantool"]])
+    -- Test that other escape symbols are not affected by the setting.
+    t.assert_equals(json.encode('\t'), [["\t"]])
+    t.assert_equals(json.encode('\\'), [["\\"]])
+
+    -- Restore options defaults.
+    compat.json_escape_forward_slash = 'default'
+end
+
+g.test_json_new_encode = function()
+    compat.json_escape_forward_slash = 'old'
+    -- Test that '/' is escaped with 'old' setting.
+    t.assert_equals(json.encode('/'), [["\/"]])
+
+    -- Create new serializer and check that it has correct defaults and doesn't change behavior.
```

Comment:
Nit: A comment is over 80 symbols long.

Reply:
It's ok in Lua. Otherwise CI would fail.

Reply:
Hm. I always think that we have 80 charachers limit. Anyway, it is not so important to spin here around.